test/base/flow.js

```diff
@@ -236,14 +246,8 @@ test("flows can be cancelled - 4 - pending Promise will be ignored", done => {
     let steps = 0
     const start = flow(function*() {
         steps = 1
-        try {
-            yield Promise.reject("This won't be catched anywhere!") // cancel will resolve this flow before this one is throw, so this promise goes uncatched
-            steps = 2
-        } catch (e) {
-            expect(steps).toBe(1)
-            expect(e.toString()).toBe("Error: FLOW_CANCELLED")
-            throw e
-        }
+        yield Promise.reject("This won't be catched anywhere!") // cancel will resolve this flow before this one is throw, so this promise goes uncatched
```

Comment:
So, this is a little risky (what happens if a cancelled function throws) but I think this is the more reasonable compromise.